frontend/src/components/transactions/TransactionDetails.tsx

```diff
@@ -16,29 +18,34 @@ export interface Props {
 }
 
 export interface State {
-  deposit: string;
+  deposit: BN;
 }
 
 export default class extends React.Component<Props, State> {
   state: State = {
-    deposit: "0"
+    deposit: new BN(0)
   };
 
   componentDidMount() {
-    this.props.transaction.actions.map(action => {
-      let actionKind: keyof T.Action;
-      let actionArgs: any;
-      if (typeof action === "string") {
-        actionKind = action;
-        actionArgs = {};
-      } else {
-        actionKind = Object.keys(action)[0] as keyof T.Action;
-        actionArgs = action[actionKind];
-      }
-      if (actionArgs.hasOwnProperty("deposit")) {
-        this.setState({ deposit: actionArgs.deposit });
-      }
-    });
+    const deposit = this.props.transaction.actions
+      .map(action => {
+        let actionKind: keyof T.Action;
+        let actionArgs: any;
+        if (typeof action === "string") {
+          actionKind = action;
+          actionArgs = {};
+        } else {
+          actionKind = Object.keys(action)[0] as keyof T.Action;
+          actionArgs = action[actionKind];
+        }
```

Comment:
Let's isolate these changes in the `src/libraries/explorer-wamp/transactions.ts`, so we avoid duplication here and in the ActionRow component.

I suggest we preprocess the actions from:

```
[actionKind[string] | {actionKind[string]: actionArgs[object]}]
```

to

```
[
{
  "kind": actionKind[string],
  "args": actionArgs[object],
}
]
```

Currently, we can have:

```js
[
      "CreateAccount",
      {
        Transfer: {
          deposit: "10000000000000000000"
        }
      },
      {
        AddKey: {
          access_key: {
            nonce: 0,
            permission: "FullAccess"
          },
          public_key: "ed25519:8LXEySyBYewiTTLxjfF1TKDsxxxxxxxxxxxxxxxxxx"
        }
      }
]
```

I want to unify it to:

```js
[
      {
        "kind": "CreateAccount",
        "args": null,
      },
      {
        "kind": "Transfer",
        "args": {
          deposit: "10000000000000000000"
        }
      },
      {
        "kind": "AddKey",
        "args": {
          access_key: {
            nonce: 0,
            permission: "FullAccess"
          },
          public_key: "ed25519:8LXEySyBYewiTTLxjfF1TKDsxxxxxxxxxxxxxxxxxx"
        }
      }
]
```

So let's do this logic right when we receive the data: https://github.com/nearprotocol/near-explorer/blob/5c5d9c3fb3cd34457c9719ae8f7123635e02c3d5/frontend/src/libraries/explorer-wamp/transactions.ts#L166